api/src/main/java/org/pmiops/workbench/db/model/DbUser.java

```diff
@@ -301,6 +301,13 @@ public void setAuthorities(Set<Short> newAuthorities) {
     this.authorities = newAuthorities;
   }
 
+  @Transient
+  public boolean hasAuthority(Authority required) {
+    // DEVELOPER is the super-authority which subsumes all others
```

Comment:
We should never have an enumeration where some states include others. This is icky. DEVELOPER should be its own, separate concept then.

Additionally, DEVELOPER is a bad name for this. I don't have or want authorities on PROD to do things like this.

Reply:
This replicates the existing Controller auth behavior.

I would support removing it, but that's going to require a separate ticket and a larger conversation at a minimum.